sdk/search/Azure.Search.Documents/tests/SessionRecords/HelloWorld/CreateIndexerAsyncAsync.json

```diff
@@ -58,7 +58,6 @@
         "x-ms-return-client-request-id": "true"
       },
       "RequestBody": {
-        "name": "gcwrdvqd",
```

Comment:
This shouldn't have been dropped, and I don't see any reason it was. The `name` parameter (and corresponding property) didn't have any changes.

Reply:
It's a readonly propery, by spec they are not be sent.

Reply:
Possibly it should be made non-readonly with a swagger workaround.

Reply:
My bad, this was part of a debugging exercise. Will revert